src/types.rs

```diff
@@ -53,6 +57,7 @@ impl std::fmt::Display for Backend {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         match self {
             Backend::FTD => write!(f, "FTD"),
+            Backend::RAW => write!(f, "RAW"),
```

Comment:
Not sure its a good idea to have different cases for `from()` and for `fmt()`. `fmt(from())` should be idempotent.